spatialprofilingtoolbox/workflow/phenotype_proximity/core.py

```diff
@@ -3,19 +3,23 @@
 """
 import warnings
 import pickle
+from typing import cast
 
 import pandas as pd
-from sklearn.neighbors import BallTree
+from pandas import DataFrame
+from sklearn.neighbors import BallTree # type: ignore
```

Comment:
Linters prefer two spaces before inline comments

Reply:
Ok, fixed. I wonder why it wasn't caught (using MyPy, Pylint, Pylance). Maybe my pylint isn't active somehow.

Reply:
autopep8 usually does this for me